### ISSUE TITLE
Prints not being serialized

### ISSUE THREAD

**Issue by gappleto97** [gappleto97/Senior-Project#6]

**Comment by gappleto97** [comment]
Current thought:

Define new print method.

Something like the following

```
print(list of stuff)
    s = ""
    for item in stuff:
        s += str(item)
    s+="/r/n"
    lock()
    sys.stdout(s)
    unlock()

**Comment by gappleto97** [comment]
Fixed in 31512538883f6dcd0b7c7bf57fe6ca1e24721031
